routers/org/home.go

```diff
@@ -93,9 +95,9 @@ func Home(ctx *context.Context) {
 	}
 
 	var opts = models.FindOrgMembersOpts{
-		OrgID:      org.ID,
-		PublicOnly: true,
-		Limit:      25,
+		OrgID:       org.ID,
+		PublicOnly:  true,
+		ListOptions: models.ListOptions{PageSize: 25},
```

Comment:
Without setting the Page here that limit does nothing.

Reply:
Actually it does because on this scenario it retrieves only the first 25 results:
https://github.com/SpaWn2KiLl/gitea/blob/list_pagination/models/org.go#L516-L518

This guarantees the exact same behavior as previous.

Reply:
Oh I see. Sorry. Probably better to stick Page: 1 in though to avoid future breakages should we refactor

Reply:
I don't know the behavior of gitea's code as good as you so I tried to change it as less as possible in order to avoid breaking stuff 😂